testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeWriteDatabricksCompatibility.java

```diff
@@ -367,7 +371,16 @@ private void testVacuumRemoveChangeDataFeedFiles(Consumer<String> vacuumExecutor
             // https://docs.delta.io/2.1.0/delta-change-data-feed.html#change-data-storage
             vacuumExecutor.accept(tableName);
 
```

Comment:
List only the files under `_change_data` . In this way you can keep the initial assertion

```
Assertions.assertThat(s3.listObjectsV2(bucketName, changeDataPrefix).getObjectSummaries()).hasSize(0);
```

```
            List<S3ObjectSummary> summaries = s3.listObjectsV2(new ListObjectsV2Request()
                            .withBucketName(bucketName)
                            .withPrefix(changeDataPrefix)
                            .withDelimiter("/"))
                    .getObjectSummaries();
```

We could use a constant for `"/"` - `PATH_SEPARATOR`

Reference implementation: 

https://github.com/trinodb/trino/blob/aa95af1188745e2eab96cb0e2600d1a79556d537/plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/TrinoS3FileSystem.java#L811-L814

Reply:
> List only the files under _change_data . In this way you can keep the initial assertion

It would be better to avoid hiding the fact of `_change_data` directory existence in my opinion. 

> We could use a constant for "/" - PATH_SEPARATOR

Disagreed. There's no benefit to replace the single character (it's a path separator obviously) with such a constant.

Reply:
Do note that keeping `_change_data` may be related to 

https://github.com/trinodb/trino/blob/513d193a2d77799d93358adfc8b5ca2e92b7b410/plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/procedure/VacuumProcedure.java#L228-L230

https://github.com/trinodb/trino/issues/12018